doc/experiment.rst

```diff
@@ -12,316 +38,211 @@ Jupyter notebook, interactive python session, or Python file and use the
+2. ``Experiment.create_model()`` :raw-html:`&rarr; ``Model``
+3. ``Experiment.create_ensemble()`` :raw-html:`&rarr; ``Ensemble``
+4. ``Experiment.create_run_settings()`` :raw-html:`&rarr; ``RunSettings``
+5. ``Experiment.create_batch_settings()`` :raw-html:`&rarr; ``BatchSettings``
+
+.. |SmartSim ERD| image:: images/edr.png
+  :width: 700
+  :alt: Alternative text
 
-    #!/bin/bash
-    #SBATCH --exclusive
-    #SBATCH --nodes=1
-    #SBATCH --ntasks-per-node=32
-    #SBATCH --time=00:10:00
+|SmartSim ERD|
+
+Moving on to the ``Model`` entity class, you will notice that a ``Model`` requires a
+minimum and maximum of one ``RunSetting`` object. However, it accepts a minimum of zero
+and a maximum of one ``BatchSetting`` object.
+
+The ``Ensemble`` entity class does not require either ``RunSetting`` or ``BatchSetting``
+objects upon initialization but does accept a maximum of one for each.
+
+Orchestrator
+^^^^^^^^^^^^
+The ``Orchestrator`` is an in-memory database that can be launched alongside
+``Model`` and ``Ensemble`` entities in SmartSim. The ``Orchestrator`` can be used to store and retrieve
+data during the course of an experiment. In order to stream data into or
+receive data from the Orchestrator, one of the SmartSim clients
+(SmartRedis) has to be used within a Model.
+
+To create a standard orchestrator, use the ``Experiment.create_database()``
+function. A standard orchestrator will run the database and applications on
+separate nodes. The database can be single or multi-sharded.
+For a colocated orchestrator, use the ``model.colocated_db()``
+function provided by the Model object. This action results in
+the launch of an orchestrator on the same node as the application
+but is limited to single-sharded database configuration.
+
+SmartSim supports multi-database functionality, enabling an experiment
+to have several concurrently launched database instances. If there is a
+need to launch more than one ``Orchestrator``, the ``Experiment.create_database()``
+function mandates the specification of a unique database identifier, denoted
+by the `db_identifier` argument, per created orchestrator.
 
-    python /path/to/script.py
+Model
+^^^^^
+Models represent any computational kernel, including applications,
+scripts, or generally, a program. They are flexible enough
+to support various applications; however, to be used with our
+clients (SmartRedis), the application must be written in Python,
+C, C++, or Fortran.
+
+A Model is created through the function ``Experiment.create_model()``.
+During initialization, models are given RunSettings objects that specify
+how a kernel should be executed with regard to the workload manager
+(e.g., Slurm) and the available compute resources on the system.
+Optionally, the user may also specify a BatchSettings object if
+the model should be launched as a batch on the WLM system.
 
 Ensemble
-========
-
-In addition to a single model, SmartSim has the ability to launch an
+^^^^^^^^
```

Comment:
This section doesn't seem to capture the essence and purpose of Ensembles.  I think we we need to add language around how it allows users to launch parameterized groups of Models and through that we explain other items like runsettings, params, etc.